examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -131,160 +143,90 @@ const PlayerSizeHeights = {
 };
 
 const DEFAULT_INITIAL_STYLES_STATE = {
-  // width: PlayerSizeWidths[MediaChromeSizes.XS] + 'px',
-};
-
-const DEFAULT_INITIAL_OPTION_STYLES_STATE = {
-  '--player-height': PlayerSizeHeights[MediaChromeSizes.LG] + 'px'
-  // '--player-height': PlayerSizeHeights[MediaChromeSizes.XS] + 'px'
+  width: `${PlayerSizeWidths[MediaChromeSizes.LG] + 1} + px`,
+  // width: PlayerSizeWidths[MediaChromeSizes.LG] + 'px',
+  // height: PlayerSizeHeights[MediaChromeSizes.LG] + 'px',
 };
 
-const reducer = (state: Partial<{ [k: string]: any }>, action): Partial<{ [k: string]: any }> => {
-  const { type, value } = action;
-  switch (type) {
-    case ActionTypes.UPDATE: {
-      return {
-        ...state,
-        ...value,
-      };
-    }
-    default: {
-      return state;
-    }
-  }
-};
-
-const toInitialState = (selectedAsset: typeof mediaAssetsJSON[0] | undefined, mediaAssets: typeof mediaAssetsJSON, query: NextParsedUrlQuery) => {
+const toInitialState = (
+  query: NextParsedUrlQuery,
+  selectedAsset?: (typeof mediaAssetsJSON)[0] | undefined,
+  mediaAssets?: typeof mediaAssetsJSON
+) => {
   const queryState = Object.fromEntries(Object.entries(query).map(([k, v]) => [k, JSON.parse(v as string)]));
   const selectedAssetState = toPlayerPropsFromJSON(selectedAsset, mediaAssets);
   const initialState = {
-    ...DEFAULT_INITIAL_STATE,
     ...selectedAssetState,
+    ...DEFAULT_INITIAL_STATE,
     ...queryState,
   };
-  return initialState;
-};
-
-const updateProps = <T extends any = any>(value: Partial<T>) => {
```

Comment:
Most of this code now exists in some form (fairly similar/identical for the most part) in `page-state/index.ts`